core/textile/model/model.go

```diff
@@ -31,21 +31,31 @@ type Model interface {
 	BucketBackupToggle(ctx context.Context, bucketSlug string, backup bool) (*BucketSchema, error)
 	FindBucket(ctx context.Context, bucketSlug string) (*BucketSchema, error)
 	ListBuckets(ctx context.Context) ([]*BucketSchema, error)
-	CreateReceivedFile(
+	CreateReceivedFileViaInvitation(
 		ctx context.Context,
 		file domain.FullPath,
 		invitationId string,
 		accepted bool,
 		key []byte,
 	) (*ReceivedFileSchema, error)
+	CreateReceivedFileViaPublicLink(
+		ctx context.Context,
+		ipfsHash string,
+		fileKey string,
```

Comment:
Noticed we are passing in the pw here so maybe we can update the name (technically doesn't make a difference since they are both strings but just in case).